app/response_processor.py

```diff
@@ -12,21 +12,24 @@
 from app import settings
 from app.helpers.exceptions import DecryptError
 
-logger = wrap_logger(logging.getLogger(__name__))
+local_logger = wrap_logger(logging.getLogger(__name__))
```

Comment:
Might be a stupid question but why do we create a local logger when we pass one into the ResponseProcessor class?

Reply:
It WAS being used in the ResponseProcessor class . Now it is only used by tests which do not pass in a logger. arguably we could remove it

Reply:
Why can't it just be called `logger`?